.admin.air.toml

```diff
@@ -0,0 +1,46 @@
+root = "."
```

Comment:
Sorry, I think this is irrelevant to this work. We already have `docker/.admin.air.toml` which serves this purpose - I thin this shouldn't be in this PR!